src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -112,6 +111,9 @@ public UIElement(string htmlTag = "div", bool isSvg = false)
 				this.Log().Debug($"Collecting UIElement for [{HtmlId}]");
 			}
 
+			// Console.WriteLine($"~{GetType()}({HtmlId}, p:{this.GetParent()})");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)